e2e/tests/update_params.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Params Namespace
+  #  TODO_DOCUMENT(@Olshansk): Document all of the on-chain governance parameters.
+
+  # TODO_TEST_IN_THIS_PR:
+  # Scenario: An unauthorized user cannot update an module params
+
+  Scenario: An authorized user updates all "tokenomics" module params
+    Given the user has the pocketd binary installed
+    And all "tokenomics" module params are set to their default values
+    And an authz grant from the "gov" "module" account to the "pnf" "user" account for the "/poktroll.tokenomics.MsgUpdateParams" message
+    When the user sends an authz exec message to update all "tokenomics" module params
+      | name                                 | value | type    |
+      | "compute_units_to_tokens_multiplier" | "420" | "int64" |
+    # Then all "tokenomics" module params should be updated
```

Comment:
@bryanchriswhite FYI, I commented these out during development and just forgot to bring it back.